keeper-gradle-plugin/src/test/kotlin/com/slack/keeper/KeeperFunctionalTest.kt

```diff
@@ -311,9 +330,11 @@ private fun buildGradleFile(testBuildType: String, automaticR8RepoManagement: Bo
   
   keeper {
     ${if (automaticR8RepoManagement) "" else "automaticR8RepoManagement = false"}
-    variantFilter {
-      setIgnore(name == "externalRelease")
-    }
+    ${if (!includeVariantFilter) "" else """
+      |  variantFilter {
```

Comment:
Nit: let’s use trimIndent() here so it doesn’t break possible syntax highlighting